src/components/ActivityCard.js

```diff
@@ -40,7 +41,7 @@ export default function ActivityCard({
             src={photo}
           />
         )}
-        {showBin !== undefined ? (
+        {showBin === true ? (
```

Comment:
`showBin` is a boolean, right? Then:
```suggestion
        {showBin ? (
```